### DISCUSSION TITLE
Source code compilation failure

### DISCUSSION THREAD

**Discussion by gggggakki** [DualSPHysics/DualSPHysics#244] [DUALSPHYSICS_V5.2]
When I download the source code written by another author from github and try to compile it, I get this type of error, I would like to know why and how to fix it, I'm not that proficient with the code. Thanks to everyone who commented!
![AXWXYR}2$_ BQA}B_G9~7GH](https://github.com/user-attachments/assets/e3615231-cfea-4552-b9f2-cbc679d75846)

**Comment by alexbexe** [comment]
written by another author? then please contact that author!
